src/components/Categories.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
 function Categories() {
+  const category = useSelector((state) => state.categories);
+  const dispatch = useDispatch();
   return (
     <div>
-      <button type="button">Check status</button>
+      <h1>{category}</h1>
+      <button type="button" onClick={() => dispatch(checkStatus())}>
```

Comment:
- [ ] @PNdunguMaina instead of calling the action creator directly in your return, kindly create a method for that, which will dispatch the action creator. Like you did in the `InputBook` component.